docs/software/sciapps/vasp.md

```diff
@@ -1,5 +1,368 @@
 [](){#ref-uenv-vasp}
 # VASP
 
-!!! todo
-    port over docs
+The Vienna Ab initio Simulation Package ([VASP]) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.
+
+VASP computes an approximate solution to the many-body Schrödinger equation, either within density functional theory (DFT), solving the Kohn-Sham equations, or within the Hartree-Fock (HF) approximation, solving the Roothaan equations.
+Hybrid functionals that mix the Hartree-Fock approach with density functional theory are implemented as well.
+Furthermore, Green's functions methods (GW quasiparticles, and ACFDT-RPA) and many-body perturbation theory (2nd-order Møller-Plesset) are available in VASP.
+
+In VASP, central quantities, like the one-electron orbitals, the electronic charge density, and the local potential are expressed in plane wave basis sets.
+The interactions between the electrons and ions are described using norm-conserving or ultrasoft pseudopotentials, or the projector-augmented-wave method.
+To determine the electronic groundstate, VASP makes use of efficient iterative matrix diagonalisation techniques, like the residual minimisation method with direct inversion of the iterative subspace (RMM-DIIS) or blocked Davidson algorithms.
+These are coupled to highly efficient Broyden and Pulay density mixing schemes to speed up the self-consistency cycle.
+
+
+!!! note "Licensing Terms and Conditions"
+    Users are kindly asked to obtain their own license. CSCS cannot provide free access to the code and needs to inform VASP Software GmbH with an updated list of users.
+    Therefore, access to precompiled VASP.6 executables and library files will be available only to users who have already purchased a VASP.6 license and upon request will become members of the CSCS unix group vasp6.
```

Comment:
```suggestion
    Therefore, access to precompiled `VASP.6` executables and library files will be available only to users who have already purchased a `VASP.6` license and upon request will become members of the CSCS unix group `vasp6`.
    
    To access VASP follow the [`Accessing Restricted Software`][ref-uenv-restricted-software] guide.
```
Do we have more details on how to become members of the `vasp6` group? (I feel we still get a considerable number of tickets about VASP access. Is there any way we can improve the situation by expanding the docs?)

Reply:
I'm not involved in the vasp6 group management, but from what I understand, Maria Grazia is the only one who can check the license. Therefore, any ticket with regards to VASP licenses should be handled by her.
Maybe we can ask the user to add a specific tag to make assignment quicker?